src/main/kotlin/lotto/domain/LottoResult.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.domain
+
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+class LottoResult(private val result: Map<Rank, Int>) {
+    fun countByRank(rank: Rank): Int {
+        return result.getOrDefault(rank, DEFAULT_VALUE)
+    }
+
+    fun calculateTotalPrize(): Money {
```

Comment:
`LottoResult` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun calculateTotalPrize(): Money {
```